content/docs/getting_started/report.md

```diff
@@ -0,0 +1,76 @@
++++
+title = "Report"
+description = "Open mSupply Reports."
+date = 2022-03-19T18:20:00+00:00
+updated = 2022-03-19T18:20:00+00:00
+draft = false
+weight = 31
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+toc = true
++++
+
+## Reports
```

Comment:
I'd suggest removing the title so that we don't have the duplication:

![image](https://github.com/user-attachments/assets/73b9b86d-0973-4ae6-bf55-7249954dcd5c)